test/test_argschema_parser.py

```diff
@@ -67,18 +66,6 @@ def test_boolean_command_line(default, args, expected):
     assert(mod.args['nest']['two'] == expected)
 
 
-def test_bad_cli_input():
```

Comment:
shouldn't this test still exist and fail a ValidationError?

Reply:
i guess its covered in your new tests...

Reply:
Yeah I removed it because of redundancy.